src/js/proxified-containers.js

```diff
@@ -50,6 +50,10 @@ proxifiedContainers = {
       return false;
     }
 
+    if (!matches.groups.port) {
+      matches.groups.port = 1080;
```

Comment:
IIRC, HTTP(S) have a different default port. Since we're accepting both SOCKS and HTTPS, we should also consider this scenario.